test/typescript-karma/Dashboards/Component/tests.js

```diff
@@ -535,3 +535,309 @@ test('DataGrid component with dataTable', async function (assert) {
+                }
+            }, {
+                id: 'micro-element',
+                type: 'JSON',
+                options: {
+                firstRowAsNames: false,
+                columnNames: ['x', 'myOpen', 'myHigh', 'myLow', 'myClose', 'mySeries1', 'mySeries2'],
+                data: [
+                    [1699434920314, 6, 5, 4, 1, 6, 9],
+                    [1699494920314, 2, 6, 2, 5, 7, 9],
+                    [1699534920314, 1, 9, 5, 3, 8, 8]
+                ]
+                }
+            }]
+        },
+        gui: {
+            layouts: [{
+                id: 'layout-1',
+                rows: [{
+                    cells: [{
+                        id: 'dashboard-col-0'
+                    }, {
+                        id: 'dashboard-col-1'
+                    }, {
+                        id: 'dashboard-col-2'
+                    }]
+                }, {
+                    cells: [{
+                        id: 'dashboard-col-3'
+                    }, {
+                        id: 'dashboard-col-4'
+                    }, {
+                        id: 'dashboard-col-5'
+                    }]
+                }]
+            }]
+        },
+        components: [{
+            cell: 'dashboard-col-0',
+            type: 'Highcharts',
+            connector: {
+                id: 'EUR-USD'
+            },
+            columnAssignment: {
+                Day: 'x',
+                EUR: 'custom.eur',
+                Rate: 'y'
+            }
+        }, {
+            cell: 'dashboard-col-1',
+            type: 'Highcharts',
+            connector: {
+                id: 'EUR-USD'
+            },
+            columnAssignment: {
+                Day: 'x',
+                EUR: 'custom.eur',
+                Rate: 'y'
+            },
+            chartOptions: {
+                yAxis: [{}, {
+                    opposite: true
+                }],
+                series: [{
+                    name: 'EUR'
+                }, {
+                    name: 'Rate',
+                    yAxis: 1
+                }]
+            }
+        }, {
+            cell: 'dashboard-col-2',
+            type: 'Highcharts',
+            connector: {
+                id: 'EUR-USD'
+            },
+            columnAssignment: {
+                Day: 'x',
+                EUR: 'custom.eur',
+                Rate: 'y'
+            },
+            chartOptions: {
+                yAxis: [{}, {
+                    opposite: true
+                }],
+                series: [{
+                    name: 'EUR',
+                    yAxis: 1
+                }]
+            }
+        }, {
+            cell: 'dashboard-col-3',
+            type: 'Highcharts',
+            connector: {
+                id: 'micro-element'
+            },
+            columnAssignment: {
+                'x': 'x',
+                'mySeries1': 'value',
+                'mySeries2': 'value',
+                seriesColumnMap: {
```

Comment:
I think maybe this should be an array, like `seriesMapping` in Highcharts - otherwise it gets very verbose, but let's discuss that in the team.

Reply:
```
 columnAssignment: {
                'x': 'x',
                'mySeries1': 'value',
                'mySeries2': 'value',
                'mySeriesName': { 
                        open: 'something',
                        high: 'somethingElse'
                }
```